packages/stream/get-stream/get-stream.test.js

```diff
@@ -0,0 +1,132 @@
+import nock from 'nock'
+import { expect } from 'chai'
+import { getStream } from './get-stream.js'
+import { Internal } from '../../internal/index.js'
+import { init } from '../../app/init/init.js'
+
+describe('Get Stream Module', function () {
+  describe('Basic test', function () {
+    it('should be a function', function () {
+      expect(getStream).to.be.a('function')
+    })
+  })
+
+  describe('Negative test', function () {
+    it('should return error response if function is called with no stream ID argument', async function () {
+      try {
+        await getStream()
+      } catch (error) {
+        expect(error).to.be.an('error')
+        expect(error.name).to.be.equal('Error')
+        expect(error.message).to.be.equal(
+          'Failed to get the stream data because the stream ID is empty. Please provide a stream ID'
+        )
+      }
+    })
+
+    it('should return error response if function is called with stream ID with not a number type', async function () {
+      try {
+        await getStream('1')
+      } catch (error) {
```

Comment:
I think we shouldn't use `try catch` for testing. We expect a function to throw an error. But that doesn't mean the error should be caught in the test file. Just let the error happens. We want to test if the function actually throws an error or not. For example, you can see the `init.test.js` file for testing reference. 

Reference: https://softwareengineering.stackexchange.com/questions/101952/should-i-use-try-catch-in-my-test-methods

Reply:
@faiq-naufal Regarding this one, I'm still confused on how to do it because not using `try catch` for testing similar to `init.test.js` because `to.throw(Error)` method is supposedly just working for sync function. However, on this getStream function I make it as async function which from what I explored, mostly they finally using try catch on testing. Or we could install chai-as-promise instead to do the testing (?) for async function.
CMIIW

Reference : 
https://github.com/chaijs/chai/issues/415
https://stackoverflow.com/questions/63511399/mocha-assert-asynchronous-function-throws-exception

Reply:
@ingrid011  Ah sorry, my bad. Thanks for reminding me again. I forgot that we were testing the asynchronous behavior for the function. `to.throw(Error)` can only work on synchronous behavior and when we test the asynchronous function we need to catch and then validate the error. I think this is already fine for me. I will check and resolve the other related with this

Reply:
@faiq-naufal that's okay, thanks for the info by the way mas. It's good learning for me. 😄